CompilerSource/compiler/compile.cpp

```diff
@@ -249,7 +249,7 @@ int lang_CPP::compile(const GameData &game, const char* exe_filename, int mode)
   }
 
   //Read the types of events
-  event_parse_resourcefile();
+  event_parse_resourcefile(game.events);
```

Comment:
And if events is empty? Do you expect people to just paste the default events.res into every game?

Reply:
afaik robert was reading events.res on file load